### DISCUSSION TITLE
How to add default hover effect to buttons for any background color

### DISCUSSION THREAD

**Discussion by KASMI7** [saadeghi/daisyui#2969] [Q&A]
let's say button has bg-[red], this button should have a hover effect just like buttons dropping their background color in daisyUI

**Comment by saadeghi** [comment]
You can use a utility class to set the hover color as well. For example `bg-red-500 hover:bg-red-600` or any other color. 

But why not use the existing button colors? They can cover almost any UI and you can change the color values from the theme.

Let me know if you have any questions

**Comment by KASMI7** [comment]
Hi saadeghi/Daisyui,

I appreciate your work, all components are looking great. I can have
custom Colors like #cc0000 which is a darker shade of red, think let's say
user-given given hex color and then the button should drop the background color
more darker than this red color like same the way you are using Oklab black
90 percent (it cannot be the black color all the time right).

I think you got my point. I think having a default hover effect is
necessary because whatever the background color it should opt for that color
and give the hover effect according to that.

Best regards.
